src/components/startworkout/WorkoutRecords.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import WeightAndRapField from "./WeightAndRapField";
+
+const WorkoutRecords = ({ setCount }: { setCount: number }) => {
```

Comment:
```suggestion
const WorkoutRecord = ({ set }: { set: number }) => {
```
1レコードのコンポーネントなので単数系がいいかなと思います

props名の修正は`WeightAndRapField`と同じ理由です！